steppers.cfg

```diff
@@ -6,85 +6,85 @@
 # In the wiring diagram, the Z-steppers are called Z1, Z2, Z3, here they're mapped to z, z1, z2 respectively.
 
 [stepper_x]
-step_pin: PE9
-dir_pin: !PF1
-enable_pin: !PF2
+step_pin: x_step_pin
+dir_pin: x_dir_pin
```

Comment:
If needed will one be able to `^x_dir_pin` to reverse?

Reply:
I think best practise would be to reverse in board_pins.cfg, as all pin settings are set there, but I see no reason why it should not work here.
I am still waiting for my hw to arrive though, so I am unable to verify.

Reply:
@pkucmus i will be sure to test this. Actually i'm kinda hoping it doesn't work, so that people are forced to do it in one place (board_pins) instead of ending up with double negations, pulled up pull downs and other interesting things 😅 it's still backwards compatible though, so that's good.